docs/BR.md

```diff
@@ -1610,9 +1611,9 @@ g. extkeyUsage (optional/required)
 >
 > This extension MUST be present and SHOULD NOT be marked critical.
 >
-> For Subordinate CA Certificates that will be used to issue TLS certificates, the value id-kp-serverAuth [RFC5280] MUST be present. The values id-kp-emailProtection [RFC5280], id-kp-codeSigning [RFC5280], id-kp-timeStamping [RFC5280], id-kp-OCSPSigning [RFC5280], and anyExtendedKeyUsage [RFC5280] MUST NOT be present. Other values SHOULD NOT be present.
+> For Subordinate CA Certificates that will be used to issue TLS certificates, the value id-kp-serverAuth [RFC5280] and/or id-kp-clientAuth [RFC5280} MUST be present. The values id-kp-emailProtection [RFC5280], id-kp-codeSigning [RFC5280], id-kp-timeStamping [RFC5280], id-kp-OCSPSigning [RFC5280], and anyExtendedKeyUsage [RFC5280] MUST NOT be present. Other values SHOULD NOT be present.
```

Comment:
So I originally tried to mirror the and/or language, but @clintwilson pointed out I kinda botched it, which you can see when I "corrected" it in https://github.com/cabforum/documents/commit/45208d90819a95affa7403ca87c8bf32ada0eaa7 and https://github.com/cabforum/documents/commit/4597b5abffa81692fd8fa08fe3b1510286564846

If I'm understanding correctly, the requirement is something like:

```suggestion
> For Subordinate CA Certificates that will be used to issue TLS certificates, the value id-kp-serverAuth [RFC5280] MUST be present. The value id-kp-clientAuth [RFC5280} MAY be present. The values id-kp-emailProtection [RFC5280], id-kp-codeSigning [RFC5280], id-kp-timeStamping [RFC5280], id-kp-OCSPSigning [RFC5280], and anyExtendedKeyUsage [RFC5280] MUST NOT be present. Other values SHOULD NOT be present.
```

This allows TLS-issuing CA to issue TLS server certificates that also contain id-kp-clientAuth (e.g. for server-to-server mTLS), while also allowing CA certificates that only issue client certificates (e.g. to smart cards) to not be subject to the TLS rules.

Does that language match the intent?